tequilapi/endpoints/transactor_test.go

```diff
@@ -82,6 +86,29 @@ func Test_Get_TransactorFees(t *testing.T) {
 	assert.JSONEq(t, mockResponse, resp.Body.String())
 }
 
+func Test_TopUp(t *testing.T) {
+	mockResponse := ""
+	server := newTestTransactorServer(http.StatusAccepted, mockResponse)
+
+	router := httprouter.New()
+
+	tr := transactor.NewTransactor(server.URL, server.URL, "0x241F6e1d0bB17f45767DC60A6Bd3D21Cdb543a0c", "0x241F6e1d0bB17f45767DC60A6Bd3D21Cdb543a0c", "0x241F6e1d0bB17f45767DC60A6Bd3D21Cdb543a0c", fakeSignerFactory)
+	AddRoutesForTransactor(router, tr)
+
+	req, err := http.NewRequest(
+		http.MethodPost,
+		"/transactor/topup",
+		bytes.NewBufferString(topUpData),
```

Comment:
As topUpData variable is used only for this test I suggest to keep it as a local variable inside this func.